packages/botonic-plugin-watson/src/watson-output-parser.js

```diff
@@ -0,0 +1,30 @@
+export default class WatsonOutputParser {
+  static UNKNOWN_INTENT_LABEL = null
+  static UNKNOWN_INTENT_CONFIDENCE = 0
+
+  static parseToBotonicFormat(output) {
+    output.intents.sort((i1, i2) => (i1.confidence > i2.confidence ? -1 : 1))
+    return {
+      intent: this.getIntentWithMaxConfidence(output.intents),
+      confidence: this.getMaxConfidence(output.intents),
+      intents: output.intents,
+      entities: this.parseEntities(output.entities),
+    }
+  }
+
+  static getIntentWithMaxConfidence(intents) {
```

Comment:
the intent with maxconfidence is the first one?

Reply:
Yes, because I first sorted them